.concourse/tasks/upgrade.sh

```diff
@@ -42,33 +72,40 @@ gcloud --quiet beta container \
   --enable-stackdriver-kubernetes \
   --enable-ip-alias \
   --network "projects/${GKE_PROJECT}/global/networks/default" \
-  --subnetwork "projects/${GKE_PROJECT}/regions/${GKE_ZONE%-?}/subnetworks/default" \
+  --subnetwork "projects/${GKE_PROJECT}/regions/${GKE_CLUSTER_ZONE%-?}/subnetworks/default" \
   --default-max-pods-per-node "110" \
   --no-enable-master-authorized-networks \
   --addons HorizontalPodAutoscaling,HttpLoadBalancing \
   --no-enable-autorepair \
   --no-enable-autoupgrade
 
-
-# Get a kubeconfig
+# Get a kubeconfig - Placed into KUBECONFIG
 gcloud container clusters get-credentials "${GKE_CLUSTER_NAME}" --zone "${GKE_CLUSTER_ZONE}" --project "${GKE_PROJECT}"
 
+# Show access settings
+cat "${KUBECONFIG}"
```

Comment:
That means we'll be deliberately printing secrets into the pipeline (and it's a secret that concourse can't hide for us because it doesn't know about it). That seems like a bad idea?

Reply:
Right, will remove that. Was needed while I was debugging.

Reply:
Done with commit 54507d8b51b33626385ae6769c4abdc6b8edcd11